src/controllers/index.controller.js

```diff
@@ -217,55 +216,25 @@ async function start(req, res) {
     }
   }
 
-  const connection = new Connection({
-    network: config.network,
-    servers: [config.server],
-    connectionTimeout: config.connectionTimeout,
-  });
-  walletConfig.connection = connection;
-
-  // tokenUid is optional but if not passed as parameter the wallet will use HTR
-  if (config.tokenUid) {
-    walletConfig.tokenUid = config.tokenUid;
-  }
-
   const preCalculatedAddresses = 'precalculatedAddresses' in req.body ? req.body.preCalculatedAddresses : [];
```

Comment:
I'd prefer not move this because this `preCalculatedAddresses` follows the same principle of the other properties on the `walletConfig` object:
- The controller checks for a property in the request
- The controller treats this property and inserts it inside the `walletConfig` object

It's different, for example, from the `Connection` object that has no input from the user http request. So it was completely moved inside the utils method.

One possibility would be to add the `preCalculatedAddresses` input to the `getWalletConfigFromSeed` and `getReadonlyWalletConfig` methods, but this would be slightly out of this PR scope. What do you think?

Reply:
When we add full support for a wallet from a list of addresses (the issue you created), then we can take care of it. For now it's ok